harness/determined/estimator/_estimator_context.py

```diff
@@ -82,12 +83,23 @@ def wrap_dataset(self, dataset: Any) -> Any:
         independently. E.g., If users instantiate their training dataset within
         ``build_train_spec()``, they should call ``dataset = wrap_dataset(dataset)``
         prior to passing it into ``tf.estimator.TrainSpec``.
+
+        Args:
+            dataset: tf.data.Dataset
+            shard_dataset: When performing multi-slot (distributed) training, this
```

Comment:
question: does this syntax render properly as restructured text?  I think in other places we use a hanging indent to distinguish between arguments and their descriptions.

Reply:
really good call, turns out the way I had it was not formatting correctly. Should be fixed now.